src/action-types/loginModalTypes.js

```diff
@@ -0,0 +1,5 @@
+const loginModalTypes = {
+  TOGGLE_LOGIN_MODAL: 'TOGGLE_LOGIN_MODAL',
+};
```

Comment:
Hey @cupidcoder, what do you think about making this action type more global in its naming? Considering so because it can be used in other modal cases such as the reset password modal.

Reply:
@Chiazokam I agree. Working on this now.

Reply:
@Chiazokam This has been worked